### PR TITLE
Adding Infiniband Dapl Tests

### PR DESCRIPTION
**Pull request by narasimhan-v**
Signed-off-by: Narasimhan V sim@linux.vnet.ibm.com
Signed-off by: Manvanthara B Puttashankar manvanth@linux.vnet.ibm.com

### CODE REVIEW

**Comment by apahim** [comment]
@narasimhan-v seems like you closed the new pull request and kept the old one open.

**Comment by narasimhan-v** [comment]
@apahim would you suggest a fresh PR or commit on top of old PR for the 1-2 line changes that was suggested in #18 ?

**Comment by apahim** [comment]
No, just close #18 and Reopen this one.

**Comment by apahim** [comment]
Ok, thank you. The code looks good. Let's just wait for someone else to provide an input here.

File: io/net/infiniband/dapl.sh.data/config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PEER_IP=15.15.15.15
```
Comment (sathnaga) [inline io/net/infiniband/dapl.sh.data/config:1]: Info on these parameters about will help users. separate readme file also would do.

File: io/net/infiniband/dapl.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
+params=(DAPL_IF1 DAPL_IF2 PEER_IP)
+for i in "${params[@]}"; do
+    eval $(cat $CONFIG_FILE | grep -w $i)
+done
+host_param=$(eval "echo $HOST_PARAM")
+peer_param=$(eval "echo $PEER_PARAM")
+
+# Timeout
+to=2m
```
Comment (sathnaga) [inline io/net/infiniband/dapl.sh]: 'to' can be expanded to 'timeout'

File: io/net/infiniband/dapl.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+   print SoftwareManager().install("dapl*")'` == 'False' ]]
+then
+    avocado_debug 'Dapl Packages not installed'
+    exit
+fi
+
+# Load Modules
+modules=(ib_uverbs ib_ucm ib_cm ib_mad ib_sa ib_umad ib_addr rdma_cm rdma_ucm \
+    ib_core mlx4_core mlx5_core mlx5_ib ib_ipoib mlx4_en mlx4_ib)
+for i in "${modules[@]}"; do
```
Comment (sathnaga) [inline io/net/infiniband/dapl.sh:19]: How about having modules list taken from config, ignore if the list might not change at all.

File: io/net/infiniband/dapl.sh.data/config (hunk 1 above)
Comment (narasimhan-v) [inline io/net/infiniband/dapl.sh.data/config:1]: Created a README file to explain these

**Comment by sathnaga** [comment]
ACKed

**Comment by apahim** [comment]
Thank you for this test.

**Pull request merged**